source/js/components/people/people.scss

```diff
@@ -8,16 +8,6 @@
     border-top: 0.25rem solid black;
   }
 
-  .person-card-featured {
-    .more-details {
-      background-color: $blue;
-      color: white;
-
-      &:hover {
-        background-color: $dark-blue;
-      }
-    }
-  }
 
```

Comment:
One more fix required: the SCSS now has two empty lines, which is an error as far as the CSS linter is concerned: when you make changes, and before you push up your commit, remember to run `npm test` to make sure there are no errors left according to the various error checked.

Reply:
Will run npm test from now on, Everything else was good?

Reply:
Submitted the change requested. Thankyou for guiding me through this. Looking forward to contribute more.

Reply:
yep, everything else seemed fine to me.